airbyte-connector-builder-server/unit_tests/connector_builder/impl/test_default_api.py

```diff
@@ -733,15 +733,15 @@ def test_resolve_manifest():
                 "requester": {
                     "path": "/v3/marketing/lists",
                     "authenticator": {"type": "BearerAuthenticator", "api_token": "{{ config.apikey }}"},
-                    "request_parameters": {"page_size": 10},
+                    "request_parameters": {"page_size": "10"},
```

Comment:
Out of curiosity, when did this change?

Reply:
This is a very interesting question. This change has been induced by https://github.com/airbytehq/airbyte/issues/19410. This was a little bit surprising to me because before this issue, I would have expected something like:
```
"requester": {
  "request_options_provider": {
    "request_parameters": {"page_size": "10"},
<...>
```

However, the `request_options_provider` seems to already be missing as if https://github.com/airbytehq/airbyte/issues/19410 was already done. So:
* Before updating the version: `requester. request_parameters` was ignored since it is not part of the schema
* After updating the version: `requester. request_parameters` started to be parsed (since we've removed the `request_option_parameters`. Since `request_parameters` expect a `Union[str, Mapping[str, str]]` and 10 was an integer and not a string